example/spec/parallel/ingestGranule/IngestGranuleFailureSpec.js

```diff
@@ -214,8 +214,8 @@ describe('The Ingest Granule failure workflow', () => {
       const granule = JSON.parse(granuleResponse.body);
 
       expect(granule.status).toBe('failed');
-      expect(granule.error.Error).toBe(syncGranFailedDetail.error);
-      expect(JSON.parse(granule.error.Cause)).toEqual(JSON.parse(syncGranFailedDetail.cause));
+      expect(granule.error.Error).toBeDefined();
```

Comment:
For posterity - per pairing - we're relying on units to cover the *specific* error here - that one exists implies a failure (1/2 possible) was recorded for purposes of the integration test.